src/class-parsely.php

```diff
@@ -2057,4 +2057,45 @@ public function parsely_is_user_logged_in() {
 	public function convert_jsonld_to_parsely_type( $type ) {
 		return in_array( $type, $this->supported_jsonld_post_types ) ? 'post' : 'index';
 	}
+
+	/**
+	 * Determine if an API key is saved in the options.
+	 *
+	 * @since 2.6.0
+	 *
+	 * @return bool True is API key is set, false if it is missing.
+	 */
+	public function api_key_is_set() {
+		$options = $this->get_options();
+
+		return (
+				isset( $options['apikey'] ) &&
+				is_string( $options['apikey'] ) &&
+				'' !== $options['apikey']
+		);
+	}
+
+	/**
+	 * Determine if an API key is not saved in the options.
+	 *
+	 * @since 2.6.0
+	 *
+	 * @return bool True if API key is missing, false if it is set.
+	 */
+	public function api_key_is_missing() {
```

Comment:
I'm not sure about this function. I think it's fine to call in our code `! $this->api_key_is_set()` and it's still very clear what it is doing.

Reply:
What's the downside of including it?

It reads more cleanly in English ("if API key is missing", as opposed to "if not API is set".

Reply:
> What's the downside of including it?

I tend to like less functions and less lines of code unless strictly necessary, and in this case we're just negating the output of a function. 

> It reads more cleanly in English ("if API key is missing", as opposed to "if not API is set".

That is true, we can keep it.